piezo_web_app/PiezoWebApp/tests/utils/route_helper_test.py

```diff
@@ -35,3 +36,27 @@ def test_is_scheme_secure_raises_value_error_for_unrecognised(scheme):
     with pytest.raises(ValueError) as exception_info:
         is_scheme_secure(scheme)
     assert str(exception_info.value) == f'"{scheme}" not recognised as a valid scheme'
+
+
+@pytest.mark.parametrize("label", ["acb123", "abc.123", "abc-123", "ab-c1.23", "a"])
```

Comment:
Would it be possible to change "label" to "name" or equivalent? It could get confusing with the regex checks for the label argument